src/shouldHideEvent.js

```diff
@@ -36,8 +36,11 @@ function _isLeaveOrJoin(ev) {
         // dupe join event, fall through to hide rules
     }
 
+
     // this only applies to joins/leaves not invites/kicks/bans
-    return membership === 'join' || (membership === 'leave' && ev.getStateKey() === ev.getSender());
+    const isJoin = membership === 'join' && prevMembership !== 'ban';
```

Comment:
This will be `true` if `prevMembership` is `invite`, so if the intention is to return `true` for invitation acceptations, then the comment above needs to be updated.